### DISCUSSION TITLE
Welcome to ALT-Scann8 Discussions!

### DISCUSSION THREAD

**Discussion by jareff-g** [jareff-g/ALT-Scann8#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clikforum** [comment]
Hi!
I'm new to GitHub, hope this is the right way to collaborate for this project.
But first I have a question: does ALT scan works with RPI-HQ Camera?

I have already builded the scan using the RPI-HQ Camera,
but It doesn't seem very "robust" to me, in fact, I would say it's rather weak,
but I think there are enough possibilities for improvement.

Congratulations on your very interesting effort

**Reply by jareff-g** [reply]
Hello @clikforum,

Thanks for your comments.

Yes, ALT-Scann8 works with the RPi-HQ camera, using the PiCamera2 library, that's the one I am using myself.

Regarding the robustness of the scanner, what do you mean exactly? If it is regarding the software and you have ideas to contribute, let me know. I'm aware it is not the best piece of code you can find, but so far I have been focused on the functionality mainly (and in learning python).

Rgds/Juan
